db/queries/create_tables.sql

```diff
@@ -29,10 +30,10 @@ CREATE TABLE proposicoes (
   data_apresentacao DATE NOT NULL,
   status JSONB NOT NULL,
   tipo_autor VARCHAR(100) NOT NULL,
-  descricao_tipo VARCHAR(100) NOT NULL,
   ementa_detalhada TEXT,
   texto TEXT,
   justificativa TEXT
+  keywords VARCHAR(500) NOT NULL,
```

Comment:
Acho desnecessária essa coluna também, mas se for ter, melhor usar `keywords TEXT ARRAY`

Reply:
Achei útil caso precisemos fazer buscas por palavras chaves. Prefere tirar?